src/test/java/nl/tudelft/pixelperfect/event/EventParametersTest.java

```diff
@@ -111,6 +122,14 @@ public void testEqualsDifferentSizes() {
   }
 
   /**
+   * An EventParameters instance should no be considered equal to something of another type.
```

Comment:
Yes, I understand, that's why you get a hundred percent. But it's about the quality of the test instead coverage, right? :)